hyperspy/tests/signal/test_eds_tem.py

```diff
@@ -135,7 +135,7 @@ def test_get_calibration_from(self):
 class Test_quantification:
 
     def setUp(self):
-        s = EDSTEMSpectrum(np.ones([2, 1024]))
+        s = EDSTEMSpectrum(np.ones([2, 2, 1024]))
         energy_axis = s.axes_manager.signal_axes[0]
```

Comment:
I agree probe_current and probe_area would make more sense to me. But I just wasn't sure if that was me being a STEM person.


> On 27 Apr 2016, at 11:36, Duncan Johnstone <notifications@github.com> wrote:
> 
> In hyperspy/tests/signal/test_eds_tem.py:
> 
> > @@ -135,7 +135,7 @@ def test_get_calibration_from(self):
> >  class Test_quantification:
> >  
> >      def setUp(self):
> > -        s = EDSTEMSpectrum(np.ones([2, 1024]))
> > +        s = EDSTEMSpectrum(np.ones([2, 2, 1024]))
> >          energy_axis = s.axes_manager.signal_axes[0]
> ok - sounds as good a reason as energy -- personally I'd like to see all of this switched to "accelerating_voltage", "probe_current", and "probe_area" -- but that is for a different PR.
> 
> —
> You are receiving this because you were mentioned.
> Reply to this email directly or view it on GitHub
>